features/zoek-met-burgerservicenummer/overzicht.feature

```diff
@@ -0,0 +1,223 @@
+    En de persoon heeft de volgende 'verblijfplaats' gegevens
+    | gemeente van inschrijving (09.10) |
+    | 0800                              |
+    Als reisdocumenten wordt gezocht met de volgende parameters
+    | naam                    | waarde                     |
+    | type                    | ZoekMetBurgerservicenummer |
+    | burgerservicenummer     | 000000152                  |
+    | gemeenteVanInschrijving | 0800                       |
+    | fields                  | reisdocumentnummer         |
+    Dan heeft de response een 'reisdocument' met de volgende gegevens
+    | naam               | waarde    |
+    | reisdocumentnummer | NE3663258 |
+
+  Scenario: de persoon heeft meerdere reisdocumenten
+    Gegeven de persoon met burgerservicenummer '000000152' heeft een 'reisdocument' met de volgende gegevens
+    | naam                                        | waarde    |
+    | soort reisdocument (35.10)                  | PN        |
+    | nummer reisdocument (35.20)                 | NE3663258 |
+    | datum einde geldigheid reisdocument (35.50) | 20240506  |
+    En de persoon heeft een 'reisdocument' met de volgende gegevens
+    | naam                                        | waarde    |
+    | soort reisdocument (35.10)                  | NI        |
+    | nummer reisdocument (35.20)                 | ID82046A7 |
+    | datum einde geldigheid reisdocument (35.50) | 20281103  |
+    En de persoon heeft de volgende 'verblijfplaats' gegevens
+    | gemeente van inschrijving (09.10) |
+    | 0800                              |
+    Als reisdocumenten wordt gezocht met de volgende parameters
+    | naam                    | waarde                     |
+    | type                    | ZoekMetBurgerservicenummer |
+    | burgerservicenummer     | 000000152                  |
+    | gemeenteVanInschrijving | 0800                       |
+    | fields                  | reisdocumentnummer         |
+    Dan heeft de response een 'reisdocument' met de volgende gegevens
+    | naam               | waarde    |
+    | reisdocumentnummer | NE3663258 |
+    En heeft de response een 'reisdocument' met de volgende gegevens
+    | naam               | waarde    |
+    | reisdocumentnummer | ID82046A7 |
+
+  Scenario: de persoon heeft geen reisdocument
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'verblijfplaats' gegevens
+    | gemeente van inschrijving (09.10) |
+    | 0800                              |
+    Als reisdocumenten wordt gezocht met de volgende parameters
+    | naam                    | waarde                     |
+    | type                    | ZoekMetBurgerservicenummer |
+    | burgerservicenummer     | 987654321                  |
+    | gemeenteVanInschrijving | 0800                       |
+    | fields                  | reisdocumentnummer,houder  |
+    Dan heeft de response 0 reisdocumenten
+
+  Scenario: zoek op een niet bestaande persoon
```

Comment:
mogelijk is het duidelijker als 'gezochte' wordt toegevoegd aan de scenarios. De titels worden dan bijv.
```
de gezochte persoon heeft één/meerdere/geen reisdocument(en)
```
Deze scenario wordt dan `de gezochte persoon bestaat niet`. De nieuwe titel triggert mij of in dit geval dan niet een 404 moet worden geretourneerd. Als consumer zie ik anders geen verschil tussen persoon bestaat niet en persoon heeft alleen ingeleverd/ingehouden/vermiste reisdocumenten

Reply:
titels wel overnemen, 404 is niet consistent.

Reply:
aangepast